### DISCUSSION TITLE
Feature request: cross-posting

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#46] [Ideas]
Hi!

I currently run two newsletters with imperfectly overlapping readerships. Would love a way to send a newsletter issue to both of them such that the overlappers only receive it once. Is that possible at all?

PS: if I had been on Buttondown when I started them I would have done this differently, but Substack's support for sub-newsletters was not good.

**Comment by jmduke** [comment]
Oh, interesting. This is a very reasonable request that I don't _think_ has a trivial way at the moment. There's some stuff we could do ad-hoc if it's just a one-off thing, but it sounds like this would be a thing that you'd want to do regularly — is that right?

(Also to ask the obvious — is there anything besides "effort" blocking you from merging them now?)

**Reply by krangor** [reply]
Yes regularly, so not worth you putting in the effort to do something bespoke esp as you're getting ready to go on leave (though the offer is appreciated)!

As to why not merge, I guess the reason I haven't considered that is that I have (and would like to retain) separate signup pages for both newsletters (so you can sub to one and not the other) and they each have their own particularly stylesheets, colour schemes, numeration, etc, which to the best of my knowledge wouldn't be separated if I used "tagging" to divvy them up? Am I wrong?

PS: extremely minor thing but I notice that on the signup pages the link colours specified in the stylesheet still aren't applied?

**Reply by jmduke** [reply]
> As to why not merge, I guess the reason I haven't considered that is that I have (and would like to retain) separate signup pages for both newsletters (so you can sub to one and not the other) and they each have their own particularly stylesheets, colour schemes, numeration, etc, which to the best of my knowledge wouldn't be separated if I used "tagging" to divvy them up? Am I wrong?

Yup, all of this is correct – and figured this was the answer :) Totally reasonable!

> PS: extremely minor thing but I notice that on the signup pages the link colours specified in the stylesheet still aren't applied?

Oh, odd. Will take a look rn.

---

Getting back to "how would we make this possible, in a generalized way?", I think the answer here is something like this:

1. We've got an [automations](https://buttondown.email/features/automation) feature that lets you do certain things when other things happen. (Tag a subscriber "foo" when they open a link, that kind of thing.)
2. We add a new automation _trigger_ that allows you to arbitrarily kick off an automation via webhook. 
3. You set up a webhook on `Newsletter A` that hits `Newsletter B` whenever `Newsletter A` gets a new subscriber.
4. `Newsletter B` has an automation that tags subscribers with `also-subscribed-to-newsletter-A`.

This is.. complicated, when I type it all out, but would get the job done.
